arrow-site/docs/docs/quickstart/README.md

```diff
@@ -7,28 +7,31 @@ permalink: /core/
 [![Kotlin version badge](https://img.shields.io/badge/Kotlin-1.4-blue)](https://kotlinlang.org/docs/reference/whatsnew14.html)
-
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://www.apache.org/licenses/LICENSE-2.0)
 [![StackOverflow](https://img.shields.io/badge/arrow--kt-grey.svg?logo=stackoverflow)](https://stackoverflow.com/questions/tagged/arrow-kt)
 [![Twitter](https://img.shields.io/twitter/follow/arrow_kt?color=blue&style=flat)](https://twitter.com/arrow_kt)
 
 Λrrow is a library for Typed Functional Programming in Kotlin.
 
-Arrow aims to provide a [*lingua franca*](https://en.wikipedia.org/wiki/Lingua_franca) of interfaces and abstractions across Kotlin libraries.
-For this, it includes the most popular data types, type classes and abstractions such as `Option`, `Either`, `Functor`, `Applicative`, `Monad` to empower users to write pure FP apps and libraries built atop higher order abstractions.
+Arrow complements the Kotlin standard library and KotlinX Coroutines libraries with functional patterns that favor composition, effect control, and immutability.
 
-Use the list below to learn more about Λrrow's main features.
+Arrow is composed of 4 main modular libraries.
 
-- [Patterns]({{ '/patterns/glossary/' | relative_url }}): tutorials and approaches to day-to-day challenges using FP
-- [Libraries]({{ '/quickstart/libraries/' | relative_url }}): all the libraries provided by Λrrow
-- [Effects]({{ '/effects/async/' | relative_url }}): interfacing with external systems
-- [Optics]({{ '/optics/iso/' | relative_url }}): inspecting and modifying data structures
+## [Core]({{ '/' | relative_url }})
+Arrow Core includes types such as [`Either`]({{ '/apidocs/arrow-core-data/arrow.core/-either/' | relative_url }}), [`Validated`]({{ '/apidocs/arrow-core-data/arrow.core/-validated/' | relative_url }}) and many extensions to [`Iterable`]({{ '/apidocs/arrow-core-data/arrow.core/kotlin.collections.-iterable/' | relative_url }}) that can be used when implementing [error handling patterns]({{ '/patterns/error_handling/' | relative_url }}).
+Core also includes the base continuation effects system which includes patterns to remove callbacks and enable controlled effects in direct syntax. Some applications of the effect system ro reduce boilerplate and enable direct syntax include [monad comprehensions and computation expressions]({{ '/patterns/monad_comprehensions/' | relative_url }}).
 
-## Curated external links
+## [Fx]({{ '/fx/' | relative_url }})
+Arrow Fx is a full-featured, high-performance, asynchronous framework that brings functional operators to Kotlin's `suspend` functions.
+By leveraging the power of KotlinX Coroutines and the compiler support for CPS transformations, Arrow Fx results in optimal async programs with increased throughput and decreased allocations friendly and in the spirit of Kotlin's coroutines and its continuations model.
```

Comment:
> increased throughput and decreased allocations

Compared to what? Do we need to specify here it's compared to other 

> decreased allocations friendly

I'm a bit confused by the friendly here 😅

Reply:
sorry I pasted bad content there last sec, updated. Feel free to make other suggestions there